Kwc/Basic/Text/Admin.php

```diff
@@ -53,6 +53,9 @@ public function duplicate($source, $target, Zend_ProgressBar $progressBar = null
             }
         }
 
+
+        $targetRow = $source->getComponent()->getOwnModel()->getRow($target->dbId);
+        if ($targetRow) { $targetRow->delete(); }
```

Comment:
mir gefällt das nicht, jetzt wird das jedesmal gelöscht, weiß aber auch grad nix besseres

Reply:
wird bei allen anderen komponenten für die own row auch in Kwc_Admin per default so gemacht